app/scripts/check_requirements

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+/**
+ * Colorize the shell output
+ *
+ * @param string $text The text.
+ * @param string $color Color of the text: green, red or nothing.
+ * @return string
+*/
+function colorizeOutput($text, $color)
+    {
+        switch ($color) {
+            case 'green':
+                // White on green
+                $output = "\033[1;37m\033[42m{$text}";
+                break;
+            case 'red':
+                // White on red
+                $output = "\033[1;37m\033[41m{$text}";
+                break;
+            default:
+                $output = $text;
+                break;
+        }
+        $output .= "\033[0m";
+        return $output;
+    }
+
+/**
+ * Check current configuration.
+ * 
+ * @param bool   $condition The type of alert.
+ * @param string $message   The message to show.
+ * @param string $help      The hint about the message.
+ * @param string $suggested_fix A message that help for fix the issue.
+ 
+ * @return void
+ */
+function check($condition, $message, $help = '', $suggested_fix = '')
+{
+    $text = "* {$message}: ";
+    if($condition) {
+        $text .= colorizeOutput($condition ? 'OK' : 'FAILED', 'green');
```

Comment:
You enter this part of the code only if $condition is true, same for the next command.

```PHP
$result = $condition ? colorizeOutput('OK', 'green') : colorizeOutput(('FAILED', 'red);
$text = "* {$message}: {$result}\n";
```